src/message/__tests__/messageActionSheet-test.js

```diff
@@ -58,6 +58,34 @@ describe('constructActionButtons', () => {
 });
 
 describe('constructTopicActionButtons', () => {
+  test('show mark as read if topic is unread', () => {
```

Comment:
You should add the tests in the same commit that you add the code in, in general.

Reply:
Sure, squashed into previous commit in the new revision.